### DISCUSSION TITLE
Sponsor request

### DISCUSSION THREAD

**Discussion by webfansplz** [community/community#66026] [Other Feature Feedback, Questions, & Ideas] [answered]
I need to apply for GitHub Sponsor, which has been under review for more than a month, please help me 🙏.

My Location: HongKong.

GitHub Username: webfansplz

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @webfansplz , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by anjuan** [comment]
Hi, @webfansplz I'm the Engineering Manager for GitHub Sponsors. I know it can be frustrating to wait for approval. In order to maintain the operational safety of Sponsors, we have to run a series of checks on each submitted profile. I'll see what I can do.

**Comment by DHayes10** [comment, marked as answer]
@webfansplz I have approved your profile. Please let me know if there is anything else you need. Thanks for reaching out.
